themes/theme-ios11/components/Headline/index.jsx

```diff
@@ -1,18 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Ellipsis from '@shopgate/pwa-common/components/Ellipsis';
+import I18n from '@shopgate/pwa-common/components/I18n';
 import styles from './style';
 
 /**
  * The headline component.
- * @param {string} props The component props.
+ * @param {Object} props The component props.
  * @returns {JSX}
  */
-const Headline = ({ text }) => (
-  text.length ? <h3 className={styles.headline}>{text}</h3> : null
-);
+const Headline = (props) => {
+  const className = props.small ? styles.small : styles.large;
+  const Component = props.small ? 'h2' : 'h1';
+
+  const hasContent = props.children || (props.text && props.text.length);
```

Comment:
Please make this an `if` condition and return `null` immediately. For better readability.